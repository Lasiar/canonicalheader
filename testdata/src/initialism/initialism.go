// Code generated by initialismer; DO NOT EDIT.
package initialism

import "net/http"

func _() {
	h := http.Header{}

	h.Get("A-Im")                              // want `use "A-IM" instead of "A-Im"`
	h.Get("Accept-Ch")                         // want `use "Accept-CH" instead of "Accept-Ch"`
	h.Get("Alpn")                              // want `use "ALPN" instead of "Alpn"`
	h.Get("Amp-Cache-Transform")               // want `use "AMP-Cache-Transform" instead of "Amp-Cache-Transform"`
	h.Get("C-Pep")                             // want `use "C-PEP" instead of "C-Pep"`
	h.Get("C-Pep-Info")                        // want `use "C-PEP-Info" instead of "C-Pep-Info"`
	h.Get("Cal-Managed-Id")                    // want `use "Cal-Managed-ID" instead of "Cal-Managed-Id"`
	h.Get("Caldav-Timezones")                  // want `use "CalDAV-Timezones" instead of "Caldav-Timezones"`
	h.Get("Cdn-Cache-Control")                 // want `use "CDN-Cache-Control" instead of "Cdn-Cache-Control"`
	h.Get("Cdn-Loop")                          // want `use "CDN-Loop" instead of "Cdn-Loop"`
	h.Get("Content-Id")                        // want `use "Content-ID" instead of "Content-Id"`
	h.Get("Content-Md5")                       // want `use "Content-MD5" instead of "Content-Md5"`
	h.Get("Dasl")                              // want `use "DASL" instead of "Dasl"`
	h.Get("Dav")                               // want `use "DAV" instead of "Dav"`
	h.Get("Differential-Id")                   // want `use "Differential-ID" instead of "Differential-Id"`
	h.Get("Dnt")                               // want `use "DNT" instead of "Dnt"`
	h.Get("Dpop")                              // want `use "DPoP" instead of "Dpop"`
	h.Get("Dpop-Nonce")                        // want `use "DPoP-Nonce" instead of "Dpop-Nonce"`
	h.Get("Ediint-Features")                   // want `use "EDIINT-Features" instead of "Ediint-Features"`
	h.Get("Etag")                              // want `use "ETag" instead of "Etag"`
	h.Get("Expect-Ct")                         // want `use "Expect-CT" instead of "Expect-Ct"`
	h.Get("Getprofile")                        // want `use "GetProfile" instead of "Getprofile"`
	h.Get("Http2-Settings")                    // want `use "HTTP2-Settings" instead of "Http2-Settings"`
	h.Get("Im")                                // want `use "IM" instead of "Im"`
	h.Get("Include-Referred-Token-Binding-Id") // want `use "Include-Referred-Token-Binding-ID" instead of "Include-Referred-Token-Binding-Id"`
	h.Get("Last-Event-Id")                     // want `use "Last-Event-ID" instead of "Last-Event-Id"`
	h.Get("Mime-Version")                      // want `use "MIME-Version" instead of "Mime-Version"`
	h.Get("Nel")                               // want `use "NEL" instead of "Nel"`
	h.Get("Odata-Entityid")                    // want `use "OData-EntityId" instead of "Odata-Entityid"`
	h.Get("Odata-Isolation")                   // want `use "OData-Isolation" instead of "Odata-Isolation"`
	h.Get("Odata-Maxversion")                  // want `use "OData-MaxVersion" instead of "Odata-Maxversion"`
	h.Get("Odata-Version")                     // want `use "OData-Version" instead of "Odata-Version"`
	h.Get("Optional-Www-Authenticate")         // want `use "Optional-WWW-Authenticate" instead of "Optional-Www-Authenticate"`
	h.Get("Oscore")                            // want `use "OSCORE" instead of "Oscore"`
	h.Get("Oslc-Core-Version")                 // want `use "OSLC-Core-Version" instead of "Oslc-Core-Version"`
	h.Get("P3p")                               // want `use "P3P" instead of "P3p"`
	h.Get("Pep")                               // want `use "PEP" instead of "Pep"`
	h.Get("Pep-Info")                          // want `use "PEP-Info" instead of "Pep-Info"`
	h.Get("Pics-Label")                        // want `use "PICS-Label" instead of "Pics-Label"`
	h.Get("Profileobject")                     // want `use "ProfileObject" instead of "Profileobject"`
	h.Get("Repeatability-Client-Id")           // want `use "Repeatability-Client-ID" instead of "Repeatability-Client-Id"`
	h.Get("Repeatability-Request-Id")          // want `use "Repeatability-Request-ID" instead of "Repeatability-Request-Id"`
	h.Get("Sec-Gpc")                           // want `use "Sec-GPC" instead of "Sec-Gpc"`
	h.Get("Sec-Websocket-Accept")              // want `use "Sec-WebSocket-Accept" instead of "Sec-Websocket-Accept"`
	h.Get("Sec-Websocket-Extensions")          // want `use "Sec-WebSocket-Extensions" instead of "Sec-Websocket-Extensions"`
	h.Get("Sec-Websocket-Key")                 // want `use "Sec-WebSocket-Key" instead of "Sec-Websocket-Key"`
	h.Get("Sec-Websocket-Protocol")            // want `use "Sec-WebSocket-Protocol" instead of "Sec-Websocket-Protocol"`
	h.Get("Sec-Websocket-Version")             // want `use "Sec-WebSocket-Version" instead of "Sec-Websocket-Version"`
	h.Get("Setprofile")                        // want `use "SetProfile" instead of "Setprofile"`
	h.Get("Slug")                              // want `use "SLUG" instead of "Slug"`
	h.Get("Soapaction")                        // want `use "SoapAction" instead of "Soapaction"`
	h.Get("Status-Uri")                        // want `use "Status-URI" instead of "Status-Uri"`
	h.Get("Tcn")                               // want `use "TCN" instead of "Tcn"`
	h.Get("Te")                                // want `use "TE" instead of "Te"`
	h.Get("Ttl")                               // want `use "TTL" instead of "Ttl"`
	h.Get("Uri")                               // want `use "URI" instead of "Uri"`
	h.Get("Www-Authenticate")                  // want `use "WWW-Authenticate" instead of "Www-Authenticate"`
	h.Get("X-Correlation-Id")                  // want `use "X-Correlation-ID" instead of "X-Correlation-Id"`
	h.Get("X-Dns-Prefetch-Control")            // want `use "X-DNS-Prefetch-Control" instead of "X-Dns-Prefetch-Control"`
	h.Get("X-Real-Ip")                         // want `use "X-Real-IP" instead of "X-Real-Ip"`
	h.Get("X-Request-Id")                      // want `use "X-Request-ID" instead of "X-Request-Id"`
	h.Get("X-Ua-Compatible")                   // want `use "X-UA-Compatible" instead of "X-Ua-Compatible"`
	h.Get("X-Webkit-Csp")                      // want `use "X-WebKit-CSP" instead of "X-Webkit-Csp"`
	h.Get("X-Xss")                             // want `use "X-XSS" instead of "X-Xss"`
	h.Get("X-Xss-Protection")                  // want `use "X-XSS-Protection" instead of "X-Xss-Protection"`
}
