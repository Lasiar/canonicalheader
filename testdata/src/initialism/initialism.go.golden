// Code generated by initialismer; DO NOT EDIT.
package initialism

import "net/http"

func _() {
	h := http.Header{}

	h.Get("A-IM")                              // want `use "A-IM" instead of "A-Im"`
	h.Get("Accept-CH")                         // want `use "Accept-CH" instead of "Accept-Ch"`
	h.Get("ALPN")                              // want `use "ALPN" instead of "Alpn"`
	h.Get("AMP-Cache-Transform")               // want `use "AMP-Cache-Transform" instead of "Amp-Cache-Transform"`
	h.Get("C-PEP")                             // want `use "C-PEP" instead of "C-Pep"`
	h.Get("C-PEP-Info")                        // want `use "C-PEP-Info" instead of "C-Pep-Info"`
	h.Get("Cal-Managed-ID")                    // want `use "Cal-Managed-ID" instead of "Cal-Managed-Id"`
	h.Get("CalDAV-Timezones")                  // want `use "CalDAV-Timezones" instead of "Caldav-Timezones"`
	h.Get("CDN-Cache-Control")                 // want `use "CDN-Cache-Control" instead of "Cdn-Cache-Control"`
	h.Get("CDN-Loop")                          // want `use "CDN-Loop" instead of "Cdn-Loop"`
	h.Get("Content-ID")                        // want `use "Content-ID" instead of "Content-Id"`
	h.Get("Content-MD5")                       // want `use "Content-MD5" instead of "Content-Md5"`
	h.Get("DASL")                              // want `use "DASL" instead of "Dasl"`
	h.Get("DAV")                               // want `use "DAV" instead of "Dav"`
	h.Get("Differential-ID")                   // want `use "Differential-ID" instead of "Differential-Id"`
	h.Get("DNT")                               // want `use "DNT" instead of "Dnt"`
	h.Get("DPoP")                              // want `use "DPoP" instead of "Dpop"`
	h.Get("DPoP-Nonce")                        // want `use "DPoP-Nonce" instead of "Dpop-Nonce"`
	h.Get("EDIINT-Features")                   // want `use "EDIINT-Features" instead of "Ediint-Features"`
	h.Get("ETag")                              // want `use "ETag" instead of "Etag"`
	h.Get("Expect-CT")                         // want `use "Expect-CT" instead of "Expect-Ct"`
	h.Get("GetProfile")                        // want `use "GetProfile" instead of "Getprofile"`
	h.Get("HTTP2-Settings")                    // want `use "HTTP2-Settings" instead of "Http2-Settings"`
	h.Get("IM")                                // want `use "IM" instead of "Im"`
	h.Get("Include-Referred-Token-Binding-ID") // want `use "Include-Referred-Token-Binding-ID" instead of "Include-Referred-Token-Binding-Id"`
	h.Get("Last-Event-ID")                     // want `use "Last-Event-ID" instead of "Last-Event-Id"`
	h.Get("MIME-Version")                      // want `use "MIME-Version" instead of "Mime-Version"`
	h.Get("NEL")                               // want `use "NEL" instead of "Nel"`
	h.Get("OData-EntityId")                    // want `use "OData-EntityId" instead of "Odata-Entityid"`
	h.Get("OData-Isolation")                   // want `use "OData-Isolation" instead of "Odata-Isolation"`
	h.Get("OData-MaxVersion")                  // want `use "OData-MaxVersion" instead of "Odata-Maxversion"`
	h.Get("OData-Version")                     // want `use "OData-Version" instead of "Odata-Version"`
	h.Get("Optional-WWW-Authenticate")         // want `use "Optional-WWW-Authenticate" instead of "Optional-Www-Authenticate"`
	h.Get("OSCORE")                            // want `use "OSCORE" instead of "Oscore"`
	h.Get("OSLC-Core-Version")                 // want `use "OSLC-Core-Version" instead of "Oslc-Core-Version"`
	h.Get("P3P")                               // want `use "P3P" instead of "P3p"`
	h.Get("PEP")                               // want `use "PEP" instead of "Pep"`
	h.Get("PEP-Info")                          // want `use "PEP-Info" instead of "Pep-Info"`
	h.Get("PICS-Label")                        // want `use "PICS-Label" instead of "Pics-Label"`
	h.Get("ProfileObject")                     // want `use "ProfileObject" instead of "Profileobject"`
	h.Get("Repeatability-Client-ID")           // want `use "Repeatability-Client-ID" instead of "Repeatability-Client-Id"`
	h.Get("Repeatability-Request-ID")          // want `use "Repeatability-Request-ID" instead of "Repeatability-Request-Id"`
	h.Get("Sec-GPC")                           // want `use "Sec-GPC" instead of "Sec-Gpc"`
	h.Get("Sec-WebSocket-Accept")              // want `use "Sec-WebSocket-Accept" instead of "Sec-Websocket-Accept"`
	h.Get("Sec-WebSocket-Extensions")          // want `use "Sec-WebSocket-Extensions" instead of "Sec-Websocket-Extensions"`
	h.Get("Sec-WebSocket-Key")                 // want `use "Sec-WebSocket-Key" instead of "Sec-Websocket-Key"`
	h.Get("Sec-WebSocket-Protocol")            // want `use "Sec-WebSocket-Protocol" instead of "Sec-Websocket-Protocol"`
	h.Get("Sec-WebSocket-Version")             // want `use "Sec-WebSocket-Version" instead of "Sec-Websocket-Version"`
	h.Get("SetProfile")                        // want `use "SetProfile" instead of "Setprofile"`
	h.Get("SLUG")                              // want `use "SLUG" instead of "Slug"`
	h.Get("SoapAction")                        // want `use "SoapAction" instead of "Soapaction"`
	h.Get("Status-URI")                        // want `use "Status-URI" instead of "Status-Uri"`
	h.Get("TCN")                               // want `use "TCN" instead of "Tcn"`
	h.Get("TE")                                // want `use "TE" instead of "Te"`
	h.Get("TTL")                               // want `use "TTL" instead of "Ttl"`
	h.Get("URI")                               // want `use "URI" instead of "Uri"`
	h.Get("WWW-Authenticate")                  // want `use "WWW-Authenticate" instead of "Www-Authenticate"`
	h.Get("X-Correlation-ID")                  // want `use "X-Correlation-ID" instead of "X-Correlation-Id"`
	h.Get("X-DNS-Prefetch-Control")            // want `use "X-DNS-Prefetch-Control" instead of "X-Dns-Prefetch-Control"`
	h.Get("X-Real-IP")                         // want `use "X-Real-IP" instead of "X-Real-Ip"`
	h.Get("X-Request-ID")                      // want `use "X-Request-ID" instead of "X-Request-Id"`
	h.Get("X-UA-Compatible")                   // want `use "X-UA-Compatible" instead of "X-Ua-Compatible"`
	h.Get("X-WebKit-CSP")                      // want `use "X-WebKit-CSP" instead of "X-Webkit-Csp"`
	h.Get("X-XSS")                             // want `use "X-XSS" instead of "X-Xss"`
	h.Get("X-XSS-Protection")                  // want `use "X-XSS-Protection" instead of "X-Xss-Protection"`
}
